internal/cmd/rabbitmq/plans/plans.go

```diff
@@ -153,15 +153,17 @@ func outputResult(p *print.Printer, outputFormat string, plans []rabbitmq.Offeri
 		table.SetHeader("OFFERING NAME", "VERSION", "ID", "NAME", "DESCRIPTION")
 		for i := range plans {
 			o := plans[i]
-			for j := range *o.Plans {
-				plan := (*o.Plans)[j]
-				table.AddRow(
-					utils.PtrString(o.Name),
-					utils.PtrString(o.Version),
-					utils.PtrString(plan.Id),
-					utils.PtrString(plan.Name),
-					utils.PtrString(plan.Description),
-				)
+			if o.Plans != nil {
+				for j := range *o.Plans {
+					plan := (*o.Plans)[j]
+					table.AddRow(
+						utils.PtrString(o.Name),
+						utils.PtrString(o.Version),
+						utils.PtrString(plan.Id),
+						utils.PtrString(plan.Name),
+						utils.PtrString(plan.Description),
+					)
+				}
 			}
 			table.AddSeparator()
```

Comment:
Take the `table.AddSeparator()` into the if body. Otherwise if multiple plans are nil, it would add unnecessary multiple separators. 
```suggestion
			table.AddSeparator()
			}
```